cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/metrics/MetricsSearchTestRun.java

```diff
@@ -46,36 +47,44 @@ private static void setupMetrics() throws Exception {
     collector.increment("reads", 1);
     collector.increment("writes", 1);
     collector = collectionService.getCollector(MetricsScope.USER,
-                                               getFlowletContext("WCount", "WordCounter", "splitter"));
+                                               getFlowletContext(Constants.DEFAULT_NAMESPACE, "WCount", "WordCounter",
+                                                                 "splitter"));
     collector.increment("reads", 1);
     collector = collectionService.getCollector(MetricsScope.USER,
-                                               getFlowletContext("WCount", "WCounter", "splitter"));
+                                               getFlowletContext(Constants.DEFAULT_NAMESPACE, "WCount", "WCounter",
+                                                                 "splitter"));
     collector.increment("reads", 1);
     collector = collectionService.getCollector(MetricsScope.USER,
-                                               getFlowletContext("WCount", "WCounter", "counter"));
+                                               getFlowletContext(Constants.DEFAULT_NAMESPACE, "WCount", "WCounter",
+                                                                 "counter"));
     collector.increment("reads", 1);
     collector = collectionService.getCollector(MetricsScope.USER,
-                                               getProcedureContext("WCount", "RCounts"));
+                                               getProcedureContext(Constants.DEFAULT_NAMESPACE, "WCount", "RCounts"));
     collector.increment("reads", 1);
     collector = collectionService.getCollector(MetricsScope.USER,
-                                               getMapReduceTaskContext("WCount", "ClassicWordCount",
+                                               getMapReduceTaskContext(Constants.DEFAULT_NAMESPACE, "WCount",
+                                                                       "ClassicWordCount",
                                                                        MapReduceMetrics.TaskType.Mapper));
     collector.increment("reads", 1);
     collector = collectionService.getCollector(MetricsScope.USER,
-                                               getMapReduceTaskContext("WCount", "ClassicWordCount",
+                                               getMapReduceTaskContext(Constants.DEFAULT_NAMESPACE, "WCount",
+                                                                       "ClassicWordCount",
                                                                        MapReduceMetrics.TaskType.Reducer));
     collector.increment("reads", 1);
     collector = collectionService.getCollector(MetricsScope.USER,
-                                               getFlowletContext("WordCount", "WordCounter", "splitter"));
+                                               getFlowletContext(Constants.DEFAULT_NAMESPACE, "WordCount",
```

Comment:
This PR was mostly dealing with v2 APIs. So I used default everywhere. However, I agree that in the ``MetricsSearchTestRun`` tests, we should use non-default namespaces. Will fix.